cs/forms.texy

```diff
@@ -11,66 +11,57 @@ Nette\Forms výrazně usnadňují vytváření a zpracování webových formulá
 \--
 
 
-K čemu vůbec nasazovat framework na jednoduchý formulář? Vyhneme se tak celé řadě rutinních úkolů, jako je třeba psaní dvojí validace (na straně serveru a JavaScriptu), minimalizujeme pravděpodobnost vzniku chyb a bezpečnostních děr.
+Použitím Nette\Forms se vyhneme celé řadě rutinních úkolů, jako je třeba psaní dvojí validace (na straně serveru a klienta), minimalizujeme pravděpodobnost vzniku chyb a bezpečnostních děr.
 
-Nette Framework klade velký důraz na bezpečnost aplikací, a proto úzkostlivě dbá i na dobré zabezpečení formulářů. Dělá to zcela transparentně a nevyžaduje manuálně nic nastavovat. Ochrání vaše aplikace před útokem [Cross Site Scripting (XSS) |vulnerability-protection#toc-cross-site-scripting-xss] i [Cross-Site Request Forgery (CSRF)| #Obrana před Cross-Site Request Forgery (CSRF)], odfiltruje ze vstupů kontrolní znaky, ověří validitu UTF-8 kódování nebo jestli nejsou položky vybrané v select boxech podvržené atd. To zní zajímavě, pojďme to vyzkoušet!
+Nette Framework klade velký důraz na bezpečnost aplikací, a proto úzkostlivě dbá i na dobré zabezpečení formulářů. Dělá to zcela transparentně a nevyžaduje manuálně nic nastavovat. Ochrání vaše aplikace před útokem [Cross Site Scripting (XSS) |vulnerability-protection#toc-cross-site-scripting-xss] i [Cross-Site Request Forgery (CSRF)| #Obrana před Cross-Site Request Forgery (CSRF)], odfiltruje ze vstupů kontrolní znaky, ověří validitu UTF-8 kódování nebo jestli nejsou položky vybrané v select boxech podvržené atd.
 
 
 První formulář
 ==============
 
-Vytvoříme si jednoduchý registrační formulář:
+Vytvoříme si v naší aplikaci jednoduchý registrační formulář. Přidáme ho do presenteru pomocí tzv. [továrničky | presenters#tovarnicky-na-komponenty]:
 
 /--php
-require 'Nette/loader.php';
+use Nette\Application\UI;
 
-use Nette\Forms\Form;
+class HomepagePresenter extends UI\Presenter
+{
 
-$form = new Form;
+    // ...
 
-$form->addText('name', 'Jméno:');
-$form->addPassword('password', 'Heslo:');
-$form->addSubmit('send', 'Registrovat');
+	protected function createComponentRegistrationForm()
+	{
+		$form = new UI\Form;
+		$form->addText('name', 'Jméno:');
+		$form->addPassword('password', 'Heslo:');
+		$form->addSubmit('login', 'Registrovat');
+		$form->onSuccess[] = array($this, 'registrationFormSubmitted');
+		return $form;
+	}
 
-echo $form; // vykreslí formulář
+	// volá se po úspěšném odeslání formuláře
+	public function registrationFormSubmitted(UI\Form $form, $values)
```

Comment:
Držel bych stejné názvosloví jako v příkladech a sandboxu a to je abcFormSucceeded().